.rubocop.yml

```diff
@@ -0,0 +1,46 @@
+AllCops:
+  DisplayCopNames: true
+  Exclude:
+   - 'bin/*'
+   - 'db/schema.rb'
+   - 'vendor/bundle/**/*'
+   - 'spec/dummy/**/*'
+  Include:
+    - 'lib/tasks/**/*'
+    - '**/Gemfile'
+    - '**/Rakefile'
+    - '**/Guardfile'
+  TargetRubyVersion: 2.3
```

Comment:
@vfonic We'd like to support two ruby versions as in travis.yml file 
rvm:
  - 2.3.1
  - 2.2.5
Could we do that?